### PR TITLE
[OJS] #6925 Change Native Import/Export plugin so that it takes the user from the Application and not the command

### PR DESCRIPTION
**Pull request by defstat**

### CODE REVIEW

File: plugins/importexport/native/NativeImportExportPlugin.inc.php [hunk 1]
```diff
@@ -353,14 +355,10 @@ function executeCLI($scriptName, &$args) {
 
 		switch ($command) {
 			case 'import':
-				$userName = array_shift($args);
-				$user = $userDao->getByUsername($userName);
+				if (!isset($user)) {
+					echo __('plugins.importexport.common.cliError') . "\n";
+					echo __('plugins.importexport.native.error.unknownUser', array('userName' => $userName)) . "\n\n";
```
Comment (NateWr) [inline plugins/importexport/native/NativeImportExportPlugin.inc.php:360]: Is `$userName` still defined? It looks like this variable was removed.

File: plugins/importexport/native/locale/en_US/locale.po [hunk 2]
```diff
@@ -71,7 +71,7 @@ msgid "plugins.importexport.native.error.unknownSection"
 msgstr "Unknown section {$param}"
 
 msgid "plugins.importexport.native.error.unknownUser"
-msgstr "The specified user, \"{$userName}\", does not exist."
+msgstr "A user must be provided by using --user_name command parameter"
```
Comment (NateWr) [inline plugins/importexport/native/locale/en_US/locale.po:74]: Change to:

> A user must be provided by using the --user_name command parameter.

File: plugins/importexport/native/NativeImportExportPlugin.inc.php [hunk 3]
```diff
@@ -353,14 +355,10 @@ function executeCLI($scriptName, &$args) {
...
 
-				if (!$user) {
-					if ($userName != '') {
-						echo __('plugins.importexport.common.cliError') . "\n";
-						echo __('plugins.importexport.native.error.unknownUser', array('userName' => $userName)) . "\n\n";
-					}
 					$this->usage($scriptName);
```
Comment (NateWr) [inline plugins/importexport/native/NativeImportExportPlugin.inc.php:362]: Update the `plugins.importexport.native.cliUsage` locale key to include the `--user_name` parameter.

**Review by NateWr — CHANGES_REQUESTED** [review]
:+1:

File: plugins/importexport/native/NativeImportExportPlugin.inc.php (hunk 1 above)
Comment (defstat) [reply plugins/importexport/native/NativeImportExportPlugin.inc.php]: Fixed

File: plugins/importexport/native/locale/en_US/locale.po (hunk 2 above)
Comment (defstat) [reply plugins/importexport/native/locale/en_US/locale.po]: Fixed

File: plugins/importexport/native/NativeImportExportPlugin.inc.php (hunk 3 above)
Comment (defstat) [reply plugins/importexport/native/NativeImportExportPlugin.inc.php:362]: Fixed

Comment (NateWr) [reply plugins/importexport/native/NativeImportExportPlugin.inc.php:362]: Did you maybe forget to include this in the commit?

Comment (defstat) [reply plugins/importexport/native/NativeImportExportPlugin.inc.php:364]: it seems so. I have just committed the change

**Pull request merged**